src/index.ts

```diff
@@ -36,6 +37,26 @@ export async function main() {
     )
     core.endGroup()
 
+    core.startGroup("Load json files into context")
+    const files = core.getInput("json-files-to-include")
+    if (files !== "") {
```

Comment:
I would extract this into its own file, such as `loadTestData` with its own top level function that you export to call here.